onnxruntime/python/tools/quantization/static_quantize_runner.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, model_path):
         name2tensors = []
         for data_dir in data_dirs:
             name2tensor = {}
-            data_paths = [os.path.join(data_dir, a) for a in sorted(os.listdir(data_dir))]
+            data_paths = [os.path.join(data_dir, f"input_{input_idx}.pb") for input_idx in range(len(model_inputs))]
```

Comment:
Should this change also require an update to the [README](https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/quantization/README.md)?